crates/assists/src/assist_context.rs

```diff
@@ -282,12 +284,15 @@ impl AssistBuilder {
             algo::diff(&node, &new).into_text_edit(&mut self.edit);
         }
     }
+    pub(crate) fn create(&mut self, create_file: FileSystemEdit) {
```

Comment:
There's probably a better API here, which more clearly expresses the operation.

Some options are:

* rename to `crate_file` and add all required data (path&contents) as parameters, as opposed to exposing FileSystemEdit directly
* rename to more general `edit_fs`